services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py

```diff
@@ -548,4 +548,4 @@ async def get_computational_shared_data_mount_point(docker_client: Docker) -> Pa
             app_settings.SIDECAR_COMP_SERVICES_SHARED_VOLUME_NAME,
             app_settings.SIDECAR_COMP_SERVICES_SHARED_FOLDER,
         )
-        return app_settings.SIDECAR_COMP_SERVICES_SHARED_FOLDER
+        return cast(Path, app_settings.SIDECAR_COMP_SERVICES_SHARED_FOLDER)
```

Comment:
Q: would it not make sense to make ```app_settings.SIDECAR_COMP_SERVICES_SHARED_FOLDER``` a Path instead?

Reply:
ok that is a bit strange. I don't quite get why mypy flags that one. Did you try to use ```reveal_type``` ?

Reply:
reveal_type is returning for some reason: Any. I will try to have a closer look at it.